### ISSUE TITLE
SET_SCORE region action is being changed from 0 to 1 on restart

### ISSUE THREAD

**Issue by PugaBear** [AddstarMC/Minigames#100]
[Example config](http://paste.bn-mc.net/letud)

When the server reboots and the Minigames plugin loads every file, it changes the SET_SCORE value (line 27) to 1, which breaks the minigame. Setting it back to 0 makes it work again, but there is no command to do this (that I know of), so we have to manually use the GUI to fix it every time.

**Comment by jupiter1390** [comment]
SET_SCORE will always reset to 0, try to use add_score instead until the bug is fixed

**Comment by Narimm** [comment]
You will need to attach the server.log report that contains the error message or start up messages for us to attempt to review this..also attach the config that causes an error

**Comment by Narimm** [comment]
With no log and no config I dont see what I can do - I have reviewed the codebase for that section and I am not seeing the cause.

**Comment by PugaBear** [comment]
@Narimm My apologies I haven't checked this in a while. I will gather that information soon

**Comment by PugaBear** [comment]
I apologize for the delay, finals week + pending 1.12 update has kept me busy.

I have scoured the logs and found no messages relating to this bug. I have watched the files closely as the server starts; up until the time when Minigames was sending the 'Your configuration will not be supported past 1.11' messages, the config would remain what it was before the reboot. Then the SET_SCORE value would be set to 1.

The regions.yml config in question is included in the first post. The corresponding main config can be found [here](http://paste.bn-mc.net/avicu). 

Beyond that, the best I can tell you is to try it yourself. Sorry for the lack of information
